near-lake-primitives/src/types/receipts.rs

```diff
@@ -0,0 +1,169 @@
+use crate::near_indexer_primitives::{
+    types::{AccountId, Balance, Gas},
+    views, CryptoHash, IndexerExecutionOutcomeWithReceipt,
+};
+use near_crypto::PublicKey;
+use near_primitives_core::serialize::{base64_format, dec_format};
+
+#[derive(Debug, Clone)]
+pub struct ExecutedReceipt {
+    pub receipt_id: CryptoHash,
+    pub receiver_id: AccountId,
+    pub predecessor_id: AccountId,
+    pub status: ExecutionStatus,
+    pub execution_outcome_id: CryptoHash,
+    pub logs: Vec<String>,
+}
+
+impl From<&IndexerExecutionOutcomeWithReceipt> for ExecutedReceipt {
+    fn from(outcome_with_receipt: &IndexerExecutionOutcomeWithReceipt) -> Self {
+        Self {
+            receipt_id: outcome_with_receipt.receipt.receipt_id,
+            receiver_id: outcome_with_receipt.receipt.receiver_id.clone(),
+            predecessor_id: outcome_with_receipt.receipt.predecessor_id.clone(),
+            execution_outcome_id: outcome_with_receipt.execution_outcome.id,
+            logs: outcome_with_receipt
+                .execution_outcome
+                .outcome
+                .logs
+                .iter()
+                .map(Clone::clone)
+                .collect(),
+            status: (&outcome_with_receipt.execution_outcome.outcome.status).into(),
+        }
+    }
+}
+
+#[derive(Debug, Clone)]
+pub enum ExecutionStatus {
+    SuccessValue(Vec<u8>),
+    SuccessReceiptId(CryptoHash),
+    Failure(String),
```

Comment:
Should we use `Failure(TxExecutionError)`, from [`near_primitives::transaction::ExecutionStatus`](https://docs.rs/near-primitives/0.15.0/near_primitives/transaction/enum.ExecutionStatus.html) instead of `String`?

Reply:
perhaps, but I don't like it too much. I've put `String` here for now and later I'll decide what to do the `TxExecutionError`